string agent
diagnostic_msgs/KeyValue[] amounts
diagnostic_msgs/KeyValue[] goals